flake.nix

```diff
@@ -15,13 +15,16 @@
   };
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
+    #nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
+    #BUG: https://github.com/NixOS/nixpkgs/issues/235179
+    #BUG: https://github.com/NixOS/nixpkgs/issues/235526
```

Comment:
Nice work on commenting the details of cross-compilation issues this way! Excellent.

Reply:
Thank you :)